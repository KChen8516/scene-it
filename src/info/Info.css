.LogoSection {
    display: flex;
    flex-direction: column;
    align-items: center;
}

.Logo {
    margin-top: 20px;
    height: 75px;
    width: 125px;
}

.SectionHeader {
    padding-left: 10px;
    padding-right: 10px;
    border-top: 1px solid #e7e7e7;
    border-bottom: 1px solid #e7e7e7;
}

.SectionHeader h2 {
    margin: 15px 0 15px 0;
    font-style: normal;
    font-weight: 200;
    color: #bcbcbc;
}