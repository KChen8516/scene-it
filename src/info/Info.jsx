import React from 'react';
import Thanos from '../assets/thanos.jpg';
import './Info.css';

const Info = () => (
    <div className="mdc-layout-grid">
        <div className="mdc-layout-grid__inner">

            <div className="LogoSection mdc-layout-grid__cell">
                <img className="Logo" src={Thanos} alt="Scene It logo" />
                <p className="mdc-typography--subtitle1" style={{margin:0}}>Scene It</p>
                <span className="mdc-typography--caption" style={{color:'#bcbcbc', fontWeight:200}}>version 1.0</span>
            </div>

            <div className="mdc-layout-grid__cell">
                <div className="SectionHeader">
                    <h2>Credits</h2>
                </div>
            </div>

            <div className="mdc-layout-grid__cell">
                <ul className="mdc-list mdc-list--dense mdc-list--non-interactive" style={{paddingLeft:30}}>
                    <li className="mdc-list-item">
                        <i className="material-icons mdc-list-item__graphic">important_devices</i>
                        Designer: Sandy Chang
                    </li>
                    <li className="mdc-list-item">
                        <i className="material-icons mdc-list-item__graphic">build</i>
                        Engineer: Kai Chen
                    </li>
                </ul>
            </div>

        </div>
    </div>
);

export default Info;
