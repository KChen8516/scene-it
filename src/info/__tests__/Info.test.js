import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Info from "../Info";

describe("Info Component", () => {
	/**
	|--------------------------------------------------
	| This is a standard snapshot test combining Jest & Enzyme.
	| Serializing the output from shallow() helps accelerate diffing.
	|--------------------------------------------------
	*/
	it("should render properly", () => {
		// Arrange
		const wrapper = shallow(<Info />);
		// Assert
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
