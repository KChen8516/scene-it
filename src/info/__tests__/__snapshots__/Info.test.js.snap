// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info Component should render properly 1`] = `
<div
  className="mdc-layout-grid"
>
  <div
    className="mdc-layout-grid__inner"
  >
    <div
      className="LogoSection mdc-layout-grid__cell"
    >
      <img
        alt="Scene It logo"
        className="Logo"
        src="thanos.jpg"
      />
      <p
        className="mdc-typography--subtitle1"
        style={
          Object {
            "margin": 0,
          }
        }
      >
        Scene It
      </p>
      <span
        className="mdc-typography--caption"
        style={
          Object {
            "color": "#bcbcbc",
            "fontWeight": 200,
          }
        }
      >
        version 1.0
      </span>
    </div>
    <div
      className="mdc-layout-grid__cell"
    >
      <div
        className="SectionHeader"
      >
        <h2>
          Credits
        </h2>
      </div>
    </div>
    <div
      className="mdc-layout-grid__cell"
    >
      <ul
        className="mdc-list mdc-list--dense mdc-list--non-interactive"
        style={
          Object {
            "paddingLeft": 30,
          }
        }
      >
        <li
          className="mdc-list-item"
        >
          <i
            className="material-icons mdc-list-item__graphic"
          >
            important_devices
          </i>
          Designer: Sandy Chang
        </li>
        <li
          className="mdc-list-item"
        >
          <i
            className="material-icons mdc-list-item__graphic"
          >
            build
          </i>
          Engineer: Kai Chen
        </li>
      </ul>
    </div>
  </div>
</div>
`;
